### PR TITLE
Update documentation for using uber jar

### PR DESCRIPTION
**Pull request by jbkyang-nvi**

### CODE REVIEW

File: docs/inference_protocols.md [hunk 1]
```diff
@@ -136,32 +136,36 @@ These can be useful references for getting started, such as the
...
- $ mvn compile exec:java -Djavacpp.platform=linux-x86_64 -Dexec.args="<your input args>"
-```
+To use the Tritonserver Java API, you will need to have the Tritonserver library
+and dependencies installed in your enviroment. There are two ways to do this:
+
+1. Use a Tritonserver docker container
+2. Build Triton from your enviroment without Docker (not recommended)
+
+After ensuring that Tritonserver and dependencies are installed, you can run your
+java program with the Java bindings with the following steps:
```
Comment (dyastremsky) [inline docs/inference_protocols.md:146]: Nit: "Java program"

**Review by dyastremsky — APPROVED** [review]
Looks good!

File: docs/inference_protocols.md [hunk 2]
```diff
@@ -136,32 +136,36 @@ These can be useful references for getting started, such as the
...
+      $ git clone --single-branch --depth=1 -b <client repo tag>
+                     https://github.com/triton-inference-server/client.git clientrepo
+      # Run build script
+      $ source clientrepo/src/java-api-bindings/scripts/install_dependencies_and_build.sh
+      ```
+      This will install the Java bindings to `/workspace/install/java-api-bindings/tritonserver-java-bindings.jar`
+   
+   *or*
+
+   b. Copying "Uber Jar" from Triton SDK container to your enviroment
```
Comment (GuanLuo) [inline docs/inference_protocols.md:162]: Why it's called "uber jar"? Is it built with customization?

Comment (dyastremsky) [reply docs/inference_protocols.md:162]: This is a Java term. (e.g. https://maven.apache.org/plugins/maven-shade-plugin/examples/includes-excludes.html)

Definition: "An uber-JAR—also known as a fat JAR or JAR with dependencies—is a JAR file that contains not only a Java program, but embeds its dependencies as well. This means that the JAR functions as an “all-in-one” distribution of the software, without needing any other Java code. (You still need a Java runtime, and an underlying operating system, of course.)"
From: https://imagej.net/develop/uber-jars

Comment (jbkyang-nvi) [reply docs/inference_protocols.md:162]: No because its a big jar with all the dependencies

**Pull request merged**